mix.exs

```diff
@@ -99,7 +99,7 @@ defmodule Scenic.Mixfile do
       {:ex_doc, ">= 0.0.0", only: [:dev]},
       {:credo, ">= 0.0.0", only: [:dev, :test], runtime: false},
       {:excoveralls, ">= 0.0.0", only: :test, runtime: false},
-      {:inch_ex, ">= 0.0.0", only: :dev, runtime: false},
+      {:inch_ex, github: "rrrene/inch_ex", only: [:dev, :test], runtime: false},
```

Comment:
Why we point to the Github version instead of Hex?!

Reply:
@yordis No, with the latest release (`1.0.0`) published on Hex I'm getting the following error:

```
(InchEx.Docs.Retriever.Error) module Mix.Tasks.Scenic.Hash was not compiled with flag --docs
```

You can look at previous Travis CI build logs.

But, @rrrene seems to have fixed that error already on `master`, he is working on the `2.0.0` version there.

Reply:
I'm going to take this (even tho I know the inch_ex report is going to be _terrible_).

We just have to remember to switch it to 2.0.0 when rrrene releases it.